src/data-types/schemas.ts

```diff
@@ -0,0 +1,147 @@
+          title: 'Timestamp',
+          type: 'number',
+        },
+        event_timestamp_ms: {
+          $id: '#root/event/event_timestamp_ms',
+          title: 'Event_timestamp_ms',
+          type: 'number',
+        },
+        event_name: {
+          $id: '#root/event/event_name',
+          title: 'Event_Name',
+          type: 'string',
+        },
+        component_id: {
+          $id: '#root/event/component_id',
+          title: 'Component_id',
+          type: 'string',
+        },
+        extension: {
+          $id: '#root/event/extension',
+          title: 'Extension',
+          type: 'object',
+          required: ['intervention'],
+          properties: {
+            intervention: {
+              $id: '#root/event/intervention',
+              title: 'Intervention',
+              type: 'object',
+              required: ['intervention_code', 'intervention_reason'],
+              properties: {
+                intervention_code: {
+                  $id: '#root/event/intervention_code',
+                  title: 'Intervention_code',
+                  type: 'string',
+                },
+                intervention_reason: {
+                  $id: '#root/event/intervention_reason',
+                  title: 'Intervention_reason',
+                  type: 'string',
+                },
+                cms_id: {
+                  $id: '#root/event/cms_id',
+                  title: 'Cms_id',
+                  type: 'string',
+                },
+                requester_id: {
+                  $id: '#root/event/requester_id',
+                  title: 'Requester_id',
+                  type: 'string',
+                },
+                audit_level: {
+                  $id: '#root/event/audit_level',
+                  title: 'Audit_level',
+                  type: 'string',
+                },
+              },
+            },
+          },
+        },
+        extensions: {
+          $id: '#root/event/extensions',
+          title: 'Extensions',
+          type: 'object',
+          properties: {
+            levelOfConfidence: {
+              $id: '#root/event/levelOfConfidence',
+              title: 'LevelOfConfidence',
+              type: 'string',
+            },
+            ciFail: {
+              $id: '#root/event/ciFail',
+              title: 'CIFail',
+              type: 'boolean',
+            },
+            hasMitigations: {
+              $id: '#root/event/hasMitigations',
+              title: 'HasMitigations',
+              type: 'boolean',
+            },
+          },
+        },
+        user: {
+          $id: '#root/event/user',
+          title: 'User',
+          type: 'object',
+          required: ['user_id'],
+          properties: {
+            user_id: {
+              $id: '#root/event/user_id',
+              title: 'User_Id',
+              type: 'string',
+              pattern: '^[^,\\s]+$',
+              minLength: 3,
+            },
+            email: {
```

Comment:
probably should ignore these attributes below as we don't need them and add `additionalProperties: true` just so we don't have to list ones we don't need.